docs/UserGuide.md

```diff
@@ -175,7 +175,19 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.<br>
+
+**Q**: How to install Java 17<br>
+**A**: Download the Java 17 [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Then follow the [installation guide](https://docs.oracle.com/en/java/javase/17/install/overview-jdk-installation.html).<br>
+
+**Q**: How to know if you have Java 17<br>
+**A**: Open your command terminal and enter `java --version`. The first line should display `java` followed by the `version number`.<br>
```

Comment:
Add instructions on how to open command terminal:
- for windows (Win+R, cmd, enter) 
- mac (??) 
- linux (Ctrl+Alt+T)

Reply:
I have added it into the guide. Thanks for your feedback.